### PR TITLE
Feature/basic structure #PR

### PR DESCRIPTION
**Pull request by fabianomag**
Add basic structure with style and updated readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/fabianofrank/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+
```
Comment (Lameck1) [inline README.md:60]: - [ ] As part of the project requirements, pease add an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+- HTML 
+- CSS 
+- JavaScript
+- React
+- Redux
+
+## Live Demo
+
+[GitHub Link](https://fabianofrank.github.io/apple-income/)
```
Comment (Lameck1) [inline README.md:21]: - [ ] In this section of the README, we should have the following:
  - A link to the project's video presentation
  - A link to the deployed live version of the project
  Please add these to the README.
  
  ### Answer to the question you left in the Code Review Request
  I would recommend you use Netlify for deployment. It is a simpler process and this React would be very responsive on Netlify. Here is a [GUIDE](https://medium.com/geekculture/how-to-deploy-react-projects-on-netlify-28f72b4ceb8c) to help with the deployment to Netlify.

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi, @fabianofrank, 

Good job so far 👏🏿!

### Some highlights
* Correct gitflow practice ✔️ 
* Good commit messages ✔️ 
* Professional README ✔️ 
* In the project:
   * Good usage of the react-router-dom package ✔️ 
   * Clean codebase ✔️ 
   * Redux store and redux logic properly implemented ✔️  
   * All expected features of the MVP functioning as expected ✔️

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] Ass required of us in the project requirements, please add the necessary tests to assert that the MPV is working as expected. Remember they should be:
  - Unit test for the redux logic
  - Integration tests for your react components
  
- [ ] _Also remember to check the comments under the review._

### Some piece of advice
- Please don't continue working on the project while you've already submitted it for review. I have realized you've pushed new changes and linter checks are also now failing. This makes the review process messy. Any work you've submitted after the review had commenced has not been checked.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leolpaz — APPROVED** [review]
Hi @fabianofrank , 

Your project is complete!
Congratulations! 🎉🎉 

To highlight:

- Congratulations on deploying your project
- Congratulations on adding the video
- Congratulations on adding the Acknowledgements to the design's author

Now there is nothing left to do but to merge it! 🏆 





Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**